src/components/Progress/CircularProgressBase/index.tsx

```diff
@@ -0,0 +1,22 @@
+import {
+    CircularProgress as MaterialCircularProgress,
+    CircularProgressProps as MaterialCircularProgressProps,
+} from "@material-ui/core";
+
+export interface CircularProgressBaseProps extends MaterialCircularProgressProps {
+    color?: "primary" | "secondary";
+}
+

```

Comment:
In my opinion, I think it's not useful because props color of MUI Circular Process is safe to use
![image](https://user-images.githubusercontent.com/58205909/130460122-8070712b-d03f-47ea-9e78-821efa9d5175.png)

Reply:
Oh I forgot to check :)) Thanks